src/molecule_podman/playbooks/create.yml

```diff
@@ -54,6 +56,7 @@
     - name: Discover local Podman images
       containers.podman.podman_image_info:
         name: "molecule_local/{{ item.item.name }}"
```

Comment:
It shouldn't respect `CONTAINER_HOST` and add `--remote`. It should use here `podman-remote` command which will use a configured environment. To configure such environment it's on user to do, we won't guess such things in ansible podman collection.

Reply:
Please check logs of `#podman` from yesterdat. Apparently they agreed that current tool behavior is confusing and in a future version podman will know to add `--remote` when `CONTAINER_HOST` is present. It kinda does this on macos/windows already.

I still think that there is no real need to make `podman` command configurable. Even if we do it would be of too limited us. Instead we can allow a `MOLECULE_PODMAN_ARGS` which would default to none and when adding `--remote` it would have the same effect as switching to podman-remote, which is mostly a shim.

The big difference with my approach is that it will allow user to pass other arguments to podman if needed.